runtime/domain.c

```diff
@@ -1212,7 +1213,7 @@ static void* domain_thread_func(void* v)
   /* Cannot access p below here. */
 
   if (domain_self) {
-    install_backup_thread(domain_self);
+    install_backup_thread_exn(domain_self);
```

Comment:
I'm reading commit-by-commit and this looks it introduces an issue where failures in `install_backup_thread_exn` are dropped silently.